### DISCUSSION TITLE
{{< contents a-span >}} missing in the last cell?

### DISCUSSION THREAD

**Discussion by fernando-dias** [quarto-dev/quarto-cli#10660] [Q&A] [answered]
### What would you like to do?

Report an issue on quarto.org

### Description

The last cell shouldn't include {{< contents a-span >}}, like the text suggested below?

We will define some values here. [{python} "some code"]{#a-span}

The output of that inline code cell is {{< contents a-span >}}. Etc.

**Comment by mcanouil** [comment]
@fernando-dias Unfortunately, your report is not usable as you did not specify where.

**Comment by fernando-dias** [comment]
Sorry. In the last (not least) cell of this page:

https://prerelease.quarto.org/docs/authoring/contents.html

**Comment by mcanouil** [comment, marked as answer]
Follow up on:
- https://github.com/quarto-dev/quarto-cli/discussions/10637
